test/state/state.hpp

```diff
@@ -120,6 +129,7 @@ struct Transaction
     std::optional<address> to;
     intx::uint256 value;
     AccessList access_list;
+    std::vector<bytes32> blob_hashes;
```

Comment:
I think we are missing `max_fee_per_data_gas` but non of the tests seems to exercise it.